Composer/packages/client/public/react-bundle.js.LICENSE.txt

```diff
@@ -0,0 +1,14 @@
+/*
```

Comment:
Do we know why these files are being generated? @tonyanziano  any ideas?

Reply:
I noticed this on my branch as well when I rebased onto yesterday's main. I suspect there was a recent PR that went in that is somehow generating this.

Reply:
I've ignored it in another PR, but couldn't figure out why there were being generated.

Reply:
[This is the only thing I could find that might be generating it](https://webpack.js.org/plugins/terser-webpack-plugin/#extractcomments) as we do use the `Terser` webpack plugin, but I don't see this option configured, nor do I see any recent changes to the webpack configs.

Reply:
I've been getting these files cropping up too. I don't know why either.

Reply:
Might be due to this: https://github.com/microsoft/BotFramework-Composer/pull/4315/files#diff-a1975c6de2cf5658b3b91350573c8791R16

Reply:
I have confirmed that just running `yarn build:react-bundles` in the `/client/` package creates these files, but the webpack configs are [incredibly minimalistic](https://github.com/microsoft/BotFramework-Composer/blob/main/Composer/packages/client/config/webpack-react-dom.config.js) with [no plugins](https://github.com/microsoft/BotFramework-Composer/blob/main/Composer/packages/client/config/webpack-react.config.js) so I'm a little bit stumped.

Reply:
I've found the issue: https://webpack.js.org/plugins/terser-webpack-plugin/#extractcomments.

I'll fix in #4336

Reply:
I have removed them from the PR for the time being